README.rst

```diff
@@ -109,7 +111,7 @@ https://git-scm.com/docs/git-submodule
 Installing the Images
 =====================
 
-See detailed instructions in doc/howtos/image-install.rst.
+See detailed instructions in ``doc/howtos/image-install.rst``.
```

Comment:
Wouldn't it be better to introduce a cross-reference between the two docs? I.e.:
- Add `.. _image-install:` at the top of `doc/howtos/image-install.rst`
- Replace this string with `image-install`_